### ISSUE TITLE
Performance mismatch

### ISSUE THREAD

**Issue by Yzichen** [Divadi/SOLOFusion#25]
I use bs=4x8, lr=1e-4 for training and the cbgs result is:

Evaluating bboxes of pts_bbox
mAP: 0.4166
mATE: 0.5791
mASE: 0.2850
mAOE: 0.5265
mAVE: 0.2424
mAAE: 0.1887
NDS: 0.5261
Eval time: 72.3s

**Comment by Divadi** [comment]
Hmm.. I have not validated lower batch size. Prior work have shown that with lower batch size, performance may decline a bit.  Perhaps you can consider tuning the learning rate a bit - I will leave this issue open so others can comment if they face similar issues.

**Comment by Mingqj** [comment]
I also have the same issues. When I use a batch size of 32, the performance will drop about 1 pp both on NDS and mAP.
